scripts/util.py

```diff
@@ -847,10 +910,15 @@ def project_metadata(key):
     return get_metadata(key, root=f"{ROOT_URL}/project/attributes")
 
 
+# Can only be used once project_metadata is run
+universe_domain(True)
```

Comment:
Why do you call it here? Please explain more

Reply:
Removed